packages/daisyui/babel.config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "../../babel.config.json"
+}
```

Comment:
Adding blank line at end to remove the red circled dash
```suggestion
}

```

Reply:
Fighting my linter. I'll fix.